sandpack-playground/src/App.tsx

```diff
@@ -0,0 +1,27 @@
+import { Sandpack } from "@codesandbox/sandpack-react";
+import React from "react";
+
+const reactCode = `export default function App() {
+  return <img src="https://placekitten.com/190" alt="Kitten" />;
+}
+`;
+
+export const App: React.FC = () => {
+  return (
+    <div>
+      <Sandpack
+        files={{
+          "/App.js": reactCode,
+        }}
+        options={{
+          autorun: true,
+          showTabs: true,
+          showLineNumbers: true,
+          showNavigator: true,
+          editorHeight: "calc(100vh - 5px)",
```

Comment:
This is a bit of a hack to get it to full-height, not really sure how else to do this as the editor doesn't scale with the outer container even when providing 100% height.

Reply:
I can update the usage here once we merge this, there's a prop through which you can pass css classes that attach to sandpack classes